app/styles/_data-toolbar.less

```diff
@@ -7,10 +7,31 @@
  .flex(@columns: 1 1 0%);
  .flex-wrap(@wrap: wrap);
  padding: 5px 0;
+ .data-toolbar-checkbox {
```

Comment:
Can you check if `class="checkbox"` does what you want?

http://getbootstrap.com/css/#checkboxes-and-radios